### ISSUE TITLE
gradio/webui support

### ISSUE THREAD

**Issue by ekswathi** [netease-youdao/EmotiVoice#56]
any chance to have a one click installer?
something like automatic1111

**Comment by syq163** [comment]
Terrific idea! We're actively exploring solutions to develop a standalone desktop application that can be easily installed with a single click. In reference to "automatic1111," are you referring to the GitHub repository at https://github.com/AUTOMATIC1111/stable-diffusion-webui?

**Comment by ekswathi** [comment]
Yes I mean automatic1111, thank you for quick replay bro. Using gradio

**Comment by syq163** [comment]
Thank you for the information, I will look into that as soon as possible.
